src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.sql

```diff
@@ -1960,6 +1967,9 @@ BEGIN
     INNER JOIN dbo.Instance AS I
     ON XQTE.Watermark = I.Watermark
     WHERE XQTE.TagKey = @tagKey
+    ORDER BY CreatedTime ASC
```

Comment:
2 concerns:
1. should we index `CreatedTime` if sort by?
2. What if there are multiple instances with same `CreatedTime`, would that return inconsistent result?

Reply:
Looks like we should paginate on unique column https://stackoverflow.com/questions/57688411/sql-pagination-duplicates-with-ordering-by-non-unique-value, https://stackoverflow.com/questions/62133543/sql-server-paging-using-limit-and-offset-with-non-unique-ordering-column

Reply:
Propose1: use `Watermark` instead, as it always unique for single tag
Propose2: still use `CreatedTime` but combination with `Watermark`, in this way, we still need to add index for them.

Reply:
I added the index after all!